cmd/registry/cmd/check/tree/tree.go

```diff
@@ -47,7 +47,7 @@ func ListSubresources(ctx context.Context,
 	handler HandlerSet) (err error) {
 	switch name := root.(type) {
 	case names.Project:
-		err = visitor.ListProjects(ctx, adminClient, name, filter, handler.ProjectHandler())
+		err = visitor.ListProjects(ctx, adminClient, name, true, filter, handler.ProjectHandler())
```

Comment:
Answering my  own question (below), you could pass name.String() or name.ProjectID here instead of bool, and we could treat an empty string as false.